src/resolvers/cart.ts

```diff
@@ -0,0 +1,45 @@
+import { IResolvers } from 'graphql-tools'
+import Store from '../models/Store'
+import Item from '../models/Item'
+import Cart from '../models/Cart'
+
+const verifyItem = (storeItems, item) => {
+  return storeItems.includes(item)
+}
+
+export const root: IResolvers = {
+  Cart: {
+    items: root => root.items.map(itemId => Item.findById(itemId))
+  }
+}
+
+export const query = {
+  getCart: (_, { cartId }) => Cart.findById(cartId),
```

Comment:
o que tu acha de deixar só assim pra manter o padrão das queries? aí lembra de substituir no schema também
```suggestion
  cart: (_, { cartId }) => Cart.findById(cartId),
```